internal/cluster/clustersetup/workflow.go

```diff
@@ -28,7 +28,10 @@ import (
 // WorkflowName can be used to reference the cluster setup workflow.
 const WorkflowName = "cluster-setup"
 
-const DeployClusterAutoscalerActivityName = "deploy-cluster-autoscaler"
+const (
+	DeployClusterAutoscalerActivityName = "deploy-cluster-autoscaler"
+	RestoreBackupActivityName           = "restore-backup"
+)
```

Comment:
I forgot to ask last time: why aren't these bundled to the activities themselves, residing in the activity implementation file?
(that was the convention previously)

Reply:
Yes, clustersetup pkg can not depend on the pkg of the activity because it would create a circular dependency to cluster pkg.

Reply:
clustersetup pkg is referenced from cluster pkg, while restore and deployAutoscaler activities depend on cluster pkg as well.

Reply:
I think a better approach would have been extracting the common cluster instead, but I understand there is a legacy hell around this, so I see why we chose this path.